src/sample_data.rs

```diff
@@ -0,0 +1,471 @@
+use std::fs;
+use std::vec::Vec;
+use std::path::PathBuf;
+use std::collections::HashMap;
+use std::collections::HashSet;
+
+use itertools::Itertools;
+use csv::Reader;
+
+
+
+fn _product_repeat(seq: &Vec<char>, k: usize) -> Vec<String> {
```

Comment:
I don't think leading underscores are idiomatic Rust. It's already a private function
```suggestion
fn product_repeat(seq: &Vec<char>, k: usize) -> Vec<String> {
```

Reply:
true, good point